home/models.py

```diff
@@ -102,6 +110,10 @@ class Answer(models.Model):
     def __str__(self):
         return self.content
 
+    @classmethod
+    def get_answers_by_date(cls):
+        return cls.objects.order_by('-publish_date')
+
```

Comment:
I made a similar function in my PR but it's not a class method, but a model function for a question that returns QuerySet of its answers filtered by date or votes. so maybe this function will be unnecessary

Reply:
I saw your function, it returns all the answers to a particular question.
My test here is to fetch all the answers from DB and see if it matches what we expect.